package.json

```diff
@@ -30,7 +30,7 @@
     "cross-fetch": "^2.1.0",
     "eth-block-tracker": "^4.4.2",
     "eth-json-rpc-filters": "^4.2.1",
-    "eth-json-rpc-infura": "5.1.0",
+    "eth-json-rpc-infura": "^5.1.0",
```

Comment:
This change should result in a change in the lockfile as well, which should also be updated here

Reply:
Good catch: b69ce6b